tests/assertions/is_substring_of.zsh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env zunit
+
```

Comment:
In fact, it seems any wildcard on the left side of the assertion will fail, but on the right side it works as expected. I'll dig out the man pages for `test` and see if that's expected.